utopia-vscode-common/src/mailbox.ts

```diff
@@ -113,7 +118,17 @@ async function waitForPathToExist(path: string, maxWaitTime: number = 5000): Pro
   }
 }
 
+async function checkAndResetIfMailboxCleared(mailbox: Mailbox): Promise<void> {
+  const mailboxClearedAtTimestamp = await getMailboxClearedAtTimestamp(inbox)
+  if (mailboxClearedAtTimestamp > mailboxLastClearedTimestamp) {
+    lastConsumedMessage = -1
```

Comment:
could you write a comment here, explaining what is happening here? I never touched mailbox.ts and I don't quite understand the implications of lastConsumedMessage being set to -1 here

Reply:
I've just added one that should hopefully help here

Reply:
did you push it? :)